google/cloud/bigtable/table_apply_test.cc

```diff
@@ -26,29 +26,43 @@ GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
 namespace {
 
 namespace bigtable = ::google::cloud::bigtable;
-using ::google::cloud::testing_util::IsContextMDValid;
 using ::google::cloud::testing_util::StatusIs;
+using ::google::cloud::testing_util::ValidateMetadataFixture;
 using ::google::cloud::testing_util::chrono_literals::operator"" _ms;
 
-auto mock_mutate_row = [](grpc::Status const& status) {
-  return [status](grpc::ClientContext* context,
-                  google::bigtable::v2::MutateRowRequest const&,
-                  google::bigtable::v2::MutateRowResponse*) {
-    EXPECT_STATUS_OK(
-        IsContextMDValid(*context, "google.bigtable.v2.Bigtable.MutateRow",
-                         google::cloud::internal::ApiClientHeader()));
-    return status;
-  };
-};
-
 class TableApplyTest : public bigtable::testing::TableTestFixture {
- public:
+ protected:
   TableApplyTest() : TableTestFixture(CompletionQueue{}) {}
+
+  Status IsContextMDValid(grpc::ClientContext& context,
+                          std::string const& method) {
+    return validate_metadata_fixture_.IsContextMDValid(
+        context, method, google::cloud::internal::ApiClientHeader());
+  }
+
+  using MutateRowMock =
+      std::function<grpc::Status(grpc::ClientContext* context,
+                                 google::bigtable::v2::MutateRowRequest const&,
+                                 google::bigtable::v2::MutateRowResponse*)>;
+
+  MutateRowMock CreateMutateRowMock(grpc::Status const& status) {
+    return [this, status](grpc::ClientContext* context,
+                          google::bigtable::v2::MutateRowRequest const&,
+                          google::bigtable::v2::MutateRowResponse*) {
+      EXPECT_STATUS_OK(
```

Comment:
same indifferent comment about a single use `ValidateMetadataFixture fixture;`

Reply:
Same rationale, the function surrounding this is called many ("many", more than once..) times.